include/boost/json/detail/align.hpp

```diff
@@ -64,11 +58,6 @@ struct max_align_t
     std::nullptr_t w;
 };
 
-#else
-using std::align;
-using max_align_t = std::max_align_t;
-#endif
```

Comment:
Boost.JSON requires C++11.
Why do you prefer the Boost instead of the features of the C++11 standard library?